arrays.ts

```diff
@@ -60,7 +60,17 @@ export const VERBS: Verb[] = [
   'pasted',
   'assembled',
   'bapped',
-  "Blåhaj'd"
+  "Blåhaj'd",
+  { word: 'fucked', cw: 'lewd, aggressive' },
+  'added',
+  'subtracted',
+  'modded',
+  'boymodded',
+  'girlmodded',
```

Comment:
awawawa did you mean `boymoded`? :3 

```suggestion
  'boymoded',
  'girlmodded',
```

Reply:
Oh just one D? I've always spelled it with 2 ds xd but sure, will correct!